styles/style.css

```diff
@@ -18,22 +18,39 @@ body {
 }
 
 body,
-button {
+button,
+.btn{
   font-family: "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
   color: var(--grey-dark);
   font-size: 1rem;
+ 
+  
 }
 
-h1,h2,h3,h4,h5,h6 {
-  margin-top: 0;
-  margin-bottom: 0;
+.btn:hover {
+  background-color: #292b2c;
+}
+
+/*buttons*/
+.btn {
+  background: #c05326;
+  padding: 5px 10px;
+  border-radius: 5px;
+  color:#fff;
+}
+
+.btn_white{
+  background: #fff;
+  color:#c05326;
```

Comment:
Nice, good work re-using the `.btn` class and applying the `.btn_white` to modify it for the white button 👍 
Often, these modifiers classes use doubles dashes instead of underscores, like this: `btn--white`. This is a standard defined under the [BEM naming convention](https://css-tricks.com/bem-101/).